src/modules/displayMovies.js

```diff
@@ -0,0 +1,46 @@
+import { updateLikes, setLike } from './likeFunctionality.js';
+
+const displayMovies = async (movieList, appId, section) => {
+  const movieSection = document.querySelector(`#${section}-section`);
+  movieSection.innerHTML = '';
+  movieList.forEach((item) => {
+    const mainDiv = document.createElement('div');
+    mainDiv.className = 'movie-wrapper';
+    mainDiv.id = `${section}${item.id}`;
+    const details = document.createElement('div');
+    details.className = 'movie-wrapper__title';
+    const img = document.createElement('img');
+    img.className = 'movie-wrapper__img';
+    img.src = item.image.medium;
+    img.id = `img_${item.id}`;
```

Comment:
[OPTIONAL] The images are not synchronized with the movies' titles in the pop-ups. For example: If I click on `Supernatural movie` the pop-up shows the image of the `The amazing race` movie not the `Supernatural` movie. I recommend fixing that for a better user experience.